pyvista/utilities/__init__.py

```
@@ -1,77 +1,39 @@
-    CGNSReader,
-    DEMReader,
-    EnSightReader,
-    FacetReader,
-    FluentReader,
-    GIFReader,
-    GLTFReader,
-    HDFReader,
-    HDRReader,
-    JPEGReader,
-    MetaImageReader,
-    MFIXReader,
-    MultiBlockPlot3DReader,
-    NIFTIReader,
-    NRRDReader,
-    OBJReader,
-    OpenFOAMReader,
-    POpenFOAMReader,
-    Plot3DFunctionEnum,
-    Plot3DMetaReader,
-    PLYReader,
-    PNGReader,
-    PNMReader,
-    PointCellDataSelection,
-    PTSReader,
-    PVDDataSet,
-    PVDReader,
-    SegYReader,
-    SLCReader,
-    STLReader,
-    TecplotReader,
-    TIFFReader,
-    TimeReader,
-    VTKDataSetReader,
-    VTKPDataSetReader,
-    XMLImageDataReader,
-    XMLPImageDataReader,
-    XMLRectilinearGridReader,
-    XMLPRectilinearGridReader,
-    XMLUnstructuredGridReader,
-    XMLPUnstructuredGridReader,
-    XMLPolyDataReader,
-    XMLStructuredGridReader,
-    XMLMultiBlockDataReader,
-    DICOMReader,
-    XdmfReader,
-)
+"""Deprecated utilities module."""
+import importlib
+import inspect
+import warnings
+
+from pyvista.core.errors import DeprecationError, PyVistaDeprecationWarning
+
+
+def __dir__():
+    raise DeprecationError('The `pyvista.utilities` module has been deprecated.')
+
+
+def __getattr__(name):
+    # TODO: for some reason this runs twice
+    utils = importlib.import_module('pyvista.core.utilities')
+    try:
+        value = inspect.getattr_static(utils, name)
+        import_path = f'from pyvista.core.utilities import {name}'
+    except AttributeError:
+        utils = importlib.import_module('pyvista.plotting.utilities')
+        try:
+            value = inspect.getattr_static(utils, name)
+            import_path = f'from pyvista.plotting.utilities import {name}'
+        except AttributeError:
+            raise AttributeError(
+                f'Module `pyvista.utilities` has been deprecated and we could not automatically find `{name}` in `pyvista.core.utilities` or `pyvista.plotting.utilities`.'
+            )
```

I'd suppress the exception context so that users only see this latter exception.
```suggestion
            ) from None
```

---

Hmm, I think I posted this to a single commit, let me repost properly. Sorry.